docs/testing/clear-cache.md

```diff
@@ -11,6 +11,9 @@ You can remove an add-in that you've previously sideloaded on Windows, Mac, or i
 
 Additionally, if you make changes to your add-in's manifest (for example, update file names of icons or text of add-in commands), you should clear the Office cache and then re-sideload the add-in using updated manifest. Doing so will allow Office to render the add-in as it's described by the updated manifest.
 
+> [!NOTE]
```

Comment:
Would it be better to put this under an H2 instead of a note? That would make it more discoverable in the side navigation. 

![image](https://user-images.githubusercontent.com/38896772/127909277-f5b7d125-bd48-43e7-8eb0-ca6aeabcf3f7.png)

Reply:
I waffled about this. At one point, I had a note at the top of the article and also a subsection with an H2. However, I ultimately decided that since this article is titled "Clear the Office cache", and clearing the browser cache is *not* clearing the Office cache, I think including browser cache clear information as an H2 subsection is misleading. As a note at the top of the article, this content has high visibility and directs readers to the appropriate article (if necessary) before digging into Office cache clear steps.

Reply:
Fair point. I agree with you, though I worry we're favoring technical accuracy over intuitive information discovery. Luckily, we can always watch the feedback (and see the link clicks) to discover if things need to change down the line.